test/e2e/workflow/deployment_test.go

```diff
@@ -181,16 +181,26 @@ var _ = Describe("NetworkAddonsConfig", func() {
 })
 
 func testConfigCreate(configSpec opv1alpha1.NetworkAddonsConfigSpec, components []Component) {
-	CreateConfig(configSpec)
-	checkConfigChange(components)
+	checkConfigChange(components, func() {
+		CreateConfig(configSpec)
```

Comment:
I'm not sure how [CreateConfig](https://github.com/kubevirt/cluster-network-addons-operator/blob/09a72b225be8c8c671beaed8eb16d9096846f962/test/operations/operations.go#L34) is considered a "while function" (as suggested from the input parameter of checkConfigChange). can you explain?

Reply:
It is `checking config change` while `creating config`. I will add a docstring describing the functionality, but you are right, it may need a better name, I can't think of any though. Any suggestion?